lib/package-managers/npm.js

```diff
@@ -44,6 +44,30 @@ function parseJson(result, data) {
     return json;
 }
 
+/**
+ * Check if package author changed between current and next version.
+ * @param {string} packageName Name of the package
+ * @param {string} currentVersion
+ * @param {string} latestVersion
+ * @returns {Promise<boolean>} A promise that fullfills with boolean value.
+ */
+async function packageAuthorChanged(packageName, currentVersion, latestVersion) {
+    npmConfig.fullMetadata = true;
+
+    const result = await pacote.packument(packageName, npmConfig);
+    if (result.versions) {
+        let pkgVersions = Object.keys(result.versions);
+        const current = semver.maxSatisfying(pkgVersions, currentVersion);
+        const latest = semver.maxSatisfying(pkgVersions, latestVersion);
+        if (current && latest) {
+            const currentAuthor = result.versions[current].author;
+            const latestAuthor = result.versions[latest].author;
```

Comment:
This appears so simply return the `author` field in the `package.json`. If a malicious user gained control of the repo, they probably would not update the `author` field.

What about the `_npmUser` field? Would that be more secure?

Reply:
You got a point, better be safe and check npm user instead